packages/eslint-config-skuba/requireExtensions.js

```diff
@@ -108,7 +125,7 @@ function processNode(node, context, check) {
     const valueWithoutSrc = value.split('src/')[1];
     const finalPath = leadingPathToSrc.includes('/src')
       ? join(leadingPathToSrc, valueWithoutSrc)
-      : join(leadingPathToSrc, 'src', valueWithoutSrc);
+      : join(findSrc(leadingPathToSrc), 'src', valueWithoutSrc);
```

Comment:
Added some functionality to try and find the `src` folder if there isn't one in the current path. Works for most cases